### PR TITLE
Sonarr Pulls

### PR DESCRIPTION
**Pull request by bakerboy448**
#### Database Migration
YES

#### Description
Close a ton of Sonarr Pulls using various Lidarr PR as patches

#### Todos
- [X] Deconflict Patch https://github.com/lidarr/Lidarr/pull/1800 
- [X] Deconflict Patch https://github.com/lidarr/Lidarr/pull/1813
- [x] Deconflict Patch https://github.com/lidarr/Lidarr/pull/1555
- [x] Deconflict Patch https://github.com/lidarr/Lidarr/pull/1619
- [x] Review Diff and account for anything missed
- [x] Review additional changes needed
- [ ] Build
- [ ] Lint

#### Issues Fixed or Closed by this PR
- [ ] Probably need to figure these out
*

### CODE REVIEW

**Comment by RetroDadson** [comment]
lidarr/Lidarr#1619 has been deconflicted

File: src/NzbDrone.Core.Test/MediaFiles/TrackImport/Specifications/SameFileSpecificationFixture.cs [hunk 1]
```diff
@@ -91,5 +92,24 @@ public void should_be_reject_if_file_size_is_the_same()
...
             Subject.IsSatisfiedBy(_localTrack, null).Accepted.Should().BeFalse();
         }
+
+        [Test]
+        public void should_be_accepted_if_file_cannot_be_fetched()
+        {
+            _localTrack.Book = (Book)Builder<Book>.CreateListOfSize(1)
+                .TheFirst(1)
+                .With(e => e.Id = 1)
+                .With(e => e.BookFiles = new LazyLoaded<List<BookFile>>(
```
Comment (RetroDadson) [inline src/NzbDrone.Core.Test/MediaFiles/TrackImport/Specifications/SameFileSpecificationFixture.cs]: @Qstick  could you check this out for me; it was the last bit that was stopping this from building (just the solution; I am letting Azure run the tests at the moment). Not entirely sure looking at it if this actually does what I am expecting it to do... (the Lidarr equivalent test wasn't much help as track is subtly different from book)

Comment (RetroDadson) [reply src/NzbDrone.Core.Test/MediaFiles/TrackImport/Specifications/SameFileSpecificationFixture.cs]: The test does fail when I run it manually so guess I have got it wrong :)

Comment (Qstick) [reply src/NzbDrone.Core.Test/MediaFiles/TrackImport/Specifications/SameFileSpecificationFixture.cs]: Yea I'll take a look tonight

Comment (Qstick) [reply src/NzbDrone.Core.Test/MediaFiles/TrackImport/Specifications/SameFileSpecificationFixture.cs]: @robindadswell See if this helps https://github.com/Readarr/Readarr/commit/ab8c78f9aebc41edf58e4169994cedd583ec94b7

It seems to pass tests (I pushed another PR with just the first batch. 

Right now you also have 4 other tests failing (I think a few are fixed in the other branch as well if you want to pick them over) and 52 frontend issues (some things seems to be mixed up)

**Comment by bakerboy448** [comment]
To Do:

- [ ] ensure we didn't break anything with the ~100+ commits
-  [ ] set PR to close the related sonarr pulls....

File: src/NzbDrone.Core/Datastore/Migration/041_add_indexer_and_enabled_release_profiles.cs [hunk 2]
```diff
@@ -0,0 +1,15 @@
+using FluentMigrator;
+using NzbDrone.Core.Datastore.Migration.Framework;
+
+namespace NzbDrone.Core.Datastore.Migration
+{
+    [Migration(041)]
```
Comment (Qstick) [inline src/NzbDrone.Core/Datastore/Migration/041_add_indexer_and_enabled_release_profiles.cs]: Needs re-numbered and re-named to 005

File: src/NzbDrone.Core/Datastore/Migration/045_remove_chown_and_folderchmod_config.cs [hunk 3]
```diff
@@ -0,0 +1,56 @@
+using System;
+using System.Data;
+using FluentMigrator;
+using NzbDrone.Common.Extensions;
+using NzbDrone.Core.Datastore.Migration.Framework;
+
+namespace NzbDrone.Core.Datastore.Migration
+{
+    [Migration(045)]
```
Comment (Qstick) [inline src/NzbDrone.Core/Datastore/Migration/045_remove_chown_and_folderchmod_config.cs]: Needs re-numbered and re-named to 006

File: src/Readarr.Http/REST/RestModule.cs [hunk 4]
```diff
@@ -16,6 +17,20 @@ public abstract class RestModule<TResource> : NancyModule
         private const string ROOT_ROUTE = "/";
         private const string ID_ROUTE = @"/(?<id>[\d]{1,10})";
 
+        // See src/Lidarr.Api.V1/Queue/QueueModule.cs
```
Comment (Qstick) [inline src/Readarr.Http/REST/RestModule.cs]: Reference to Readarr

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -19,11 +19,12 @@ Setup guides, FAQ, the more information we have on the [wiki](https://wiki.serva
 
 1. Fork Readarr
 2. Clone the repository into your development machine. [*info*](https://help.github.com/articles/working-with-repositories)
-3. Install the required Node Packages `yarn install`
-4. Start gulp to monitor your dev environment for any changes that need post processing using `yarn start` command.
-5. Build the project in Visual Studio, Setting startup project to `Readarr.Console` and framework to `netcoreapp31`
-6. Debug the project in Visual Studio
-7. Open http://localhost:8787
+3. Grab the submodules `git submodule init && git submodule update`
```
Comment (Qstick) [inline CONTRIBUTING.md]: Ditch this, no submodules

File: azure-pipelines.yml [hunk 6]
```diff
@@ -697,17 +697,6 @@ stages:
...
-        displayName: 'Copy Screenshot to: $(Build.ArtifactStagingDirectory)'
-        inputs:
-          SourceFolder: '$(Build.SourcesDirectory)'
-          Contents: |
-            **/*_test_screenshot.png
-          TargetFolder: '$(Build.ArtifactStagingDirectory)/screenshots'
-      - publish: $(Build.ArtifactStagingDirectory)/screenshots
-        artifact: '$(osName)AutomationScreenshots'
-        condition: and(succeeded(), eq(variables['System.JobAttempt'], '1'))
-        displayName: Publish Screenshot Bundle
```
Comment (Qstick) [inline azure-pipelines.yml]: I don't think this block should be removed?

File: frontend/src/Settings/Profiles/Metadata/PrimaryTypeItem.js [hunk 7]
```diff
@@ -0,0 +1,60 @@
+import classNames from 'classnames';
```
Comment (Qstick) [inline frontend/src/Settings/Profiles/Metadata/PrimaryTypeItem.js]: I think this was pulled from Readarr as it wasn't needed?

File: frontend/src/Settings/Profiles/Metadata/PrimaryTypeItems.js [hunk 8]
```diff
@@ -0,0 +1,87 @@
+import PropTypes from 'prop-types';
```
Comment (Qstick) [inline frontend/src/Settings/Profiles/Metadata/PrimaryTypeItems.js]: I think this was pulled from Readarr as it wasn't needed?

File: frontend/src/Settings/Profiles/Metadata/ReleaseStatusItem.js [hunk 9]
```diff
@@ -0,0 +1,60 @@
+import classNames from 'classnames';
```
Comment (Qstick) [inline frontend/src/Settings/Profiles/Metadata/ReleaseStatusItem.js]: I think this was pulled from Readarr as it wasn't needed?

File: frontend/src/Settings/Profiles/Metadata/ReleaseStatusItems.js [hunk 10]
```diff
@@ -0,0 +1,87 @@
+import PropTypes from 'prop-types';
```
Comment (Qstick) [inline frontend/src/Settings/Profiles/Metadata/ReleaseStatusItems.js]: I think this was pulled from Readarr as it wasn't needed?

File: frontend/src/Settings/Profiles/Metadata/SecondaryTypeItem.js [hunk 11]
```diff
@@ -0,0 +1,60 @@
+import classNames from 'classnames';
```
Comment (Qstick) [inline frontend/src/Settings/Profiles/Metadata/SecondaryTypeItem.js]: I think this was pulled from Readarr as it wasn't needed?

File: frontend/src/Settings/Profiles/Metadata/SecondaryTypeItems.js [hunk 12]
```diff
@@ -0,0 +1,87 @@
+import PropTypes from 'prop-types';
```
Comment (Qstick) [inline frontend/src/Settings/Profiles/Metadata/SecondaryTypeItems.js]: I think this was pulled from Readarr as it wasn't needed?

File: frontend/src/Store/Actions/authorEditorActions.js [hunk 13]
```diff
@@ -30,6 +31,58 @@ export const defaultState = {
...
+    },
+    {
+      name: 'metadataProfileId',
+      label: 'Metadata Profile',
+      isSortable: true,
+      isVisible: false
+    },
+    {
+      name: 'albumFolder',
+      label: 'Album Folder',
```
Comment (Qstick) [inline frontend/src/Store/Actions/authorEditorActions.js]: Remove, AlbumFolder was Lidarr

File: frontend/src/Store/Actions/bookFileActions.js [hunk 14]
```diff
@@ -200,26 +200,30 @@ export const actionHandlers = handleThunks({
...
+      data: JSON.stringify(requestData)
     }).request;
 
-    promise.done(() => {
+    promise.done((data) => {
       dispatch(batchActions([
         ...bookFileIds.map((id) => {
           const props = {};
 
+          const trackFile = data.find((file) => file.id === id);
```
Comment (Qstick) [inline frontend/src/Store/Actions/bookFileActions.js]: Name to `bookFile`

File: frontend/src/Store/Selectors/createHealthCheckSelector.js [hunk 15]
```diff
@@ -0,0 +1,24 @@
...
+    (state) => state.app,
+    (health, app) => {
+      const items = [...health.items];
+
+      if (!app.isConnected) {
+        items.push({
+          source: 'UI',
+          type: 'warning',
+          message: 'Could not connect to SignalR, UI won\'t update',
+          wikiUrl: 'https://github.com/Readarr/Readarr/wiki/Health-Checks#could-not-connect-to-signalr'
```
Comment (Qstick) [inline frontend/src/Store/Selectors/createHealthCheckSelector.js]: Go ahead and link new wiki?

File: frontend/src/Styles/Variables/dimensions.js [hunk 16]
```diff
@@ -46,8 +46,8 @@ module.exports = {
   // Modal
   modalBodyPadding: '30px',
 
-  // Author
-  authorIndexColumnPadding: '20px',
-  authorIndexColumnPaddingSmallScreen: '10px',
+  // Artist
```
Comment (Qstick) [inline frontend/src/Styles/Variables/dimensions.js]: Author

File: src/Lidarr.Api.V1/Blacklist/BlacklistBulkResource.cs [hunk 17]
```diff
@@ -0,0 +1,9 @@
+using System.Collections.Generic;
```
Comment (Qstick) [inline src/Lidarr.Api.V1/Blacklist/BlacklistBulkResource.cs]: This file needs to be in Readarr.Api.V1 folder

File: src/Lidarr.Api.V1/Blacklist/BlacklistBulkResource.cs [hunk 18]
```diff
@@ -0,0 +1,9 @@
...
+
+namespace Lidarr.Api.V1.Blacklist
```
Comment (Qstick) [inline src/Lidarr.Api.V1/Blacklist/BlacklistBulkResource.cs]: Fix namespace

File: src/NzbDrone.Update/UpdateEngine/InstallUpdateService.cs [hunk 19]
```diff
@@ -90,6 +90,12 @@ public void Start(string installationFolder, int processId)
 
             Verify(installationFolder, processId);
 
+            if (installationFolder.EndsWith(@"\bin\Lidarr") || installationFolder.EndsWith(@"/bin/Lidarr"))
```
Comment (Qstick) [inline src/NzbDrone.Update/UpdateEngine/InstallUpdateService.cs]: Fix these two Readarr references

File: src/NzbDrone.Core/Notifications/SendGrid/SendGrid.cs [hunk 20]
```diff
@@ -0,0 +1,67 @@
...
+        }
+
+        public override ValidationResult Test()
+        {
+            var failures = new List<ValidationFailure>();
+
+            try
+            {
+                const string title = "Test Notification";
+                const string body = "This is a test message from Lidarr";
```
Comment (Qstick) [inline src/NzbDrone.Core/Notifications/SendGrid/SendGrid.cs]: Lidarr to Readarr

File: src/NzbDrone.Core/Notifications/Prowl/ProwlProxy.cs [hunk 21]
```diff
@@ -0,0 +1,78 @@
...
+                throw new ProwlException("Failed to connect to prowl, please check your settings.", ex);
+            }
+        }
+
+        public ValidationFailure Test(ProwlSettings settings)
+        {
+            try
+            {
+                const string title = "Test Notification";
+                const string body = "This is a test message from Lidarr";
```
Comment (Qstick) [inline src/NzbDrone.Core/Notifications/Prowl/ProwlProxy.cs]: Lidarr to Readarr

File: src/NzbDrone.Core/MediaFiles/BookImport/Specifications/SameFileSpecification.cs [hunk 22]
```diff
@@ -27,9 +27,17 @@ public Decision IsSatisfiedBy(LocalBook item, DownloadClientItem downloadClientI
 
             foreach (var bookFile in bookFiles)
             {
-                if (bookFile.Size == item.Size)
+                if (bookFile == null)
                 {
-                    _logger.Debug("'{0}' Has the same filesize as existing file", item.Path);
+                    var book = localBook.Book;
+                    _logger.Trace("Unable to get book file details from the DB. BookId: {0} BookFileId: {1}", book.Id, bookFile.Id);
```
Comment (Qstick) [inline src/NzbDrone.Core/MediaFiles/BookImport/Specifications/SameFileSpecification.cs]: `bookFile` is always null here, thus remove it from the log out.

File: src/NzbDrone.Core/Indexers/Newznab/NewznabCategoryFieldOptionsConverter.cs [hunk 23]
```diff
@@ -0,0 +1,65 @@
...
+                categories.Add(new NewznabCategory
+                {
+                    Id = 3000,
+                    Name = "Music",
+                    Subcategories = new List<NewznabCategory>
+                    {
+                        new NewznabCategory { Id = 3040, Name = "Loseless" },
+                        new NewznabCategory { Id = 3010, Name = "MP3" },
+                        new NewznabCategory { Id = 3050, Name = "Other" },
+                        new NewznabCategory { Id = 3030, Name = "Audiobook" }
```
Comment (Qstick) [inline src/NzbDrone.Core/Indexers/Newznab/NewznabCategoryFieldOptionsConverter.cs]: These defaults should be changed for ebooks instead of the defaults for music.

File: src/NzbDrone.Core/Indexers/Newznab/NewznabCategoryFieldOptionsConverter.cs [hunk 24]
```diff
@@ -0,0 +1,65 @@
...
+using NzbDrone.Core.Annotations;
+
+namespace NzbDrone.Core.Indexers.Newznab
+{
+    public static class NewznabCategoryFieldOptionsConverter
+    {
+        public static List<FieldSelectOption> GetFieldSelectOptions(List<NewznabCategory> categories)
+        {
+            // Ignore categories not relevant for Lidarr
+            var ignoreCategories = new[] { 1000, 2000, 4000, 5000, 6000, 7000 };
```
Comment (Qstick) [inline src/NzbDrone.Core/Indexers/Newznab/NewznabCategoryFieldOptionsConverter.cs]: Rework the ignoreCats for ebooks instead of for music.

File: src/NzbDrone.Core/Indexers/FileList/FileListSettings.cs [hunk 25]
```diff
@@ -0,0 +1,70 @@
...
+            return new NzbDroneValidationResult(Validator.Validate(this));
+        }
+    }
+
+    public enum FileListCategories
+    {
+        [FieldOption]
+        AUDIO = 5,
+        [FieldOption]
+        FLAC = 11
```
Comment (Qstick) [inline src/NzbDrone.Core/Indexers/FileList/FileListSettings.cs]: Leave audio and add ebook catecory: DOCS = 16

File: src/NzbDrone.Core/Indexers/FileList/FileListSettings.cs [hunk 26]
```diff
@@ -0,0 +1,70 @@
...
+        private static readonly FileListSettingsValidator Validator = new FileListSettingsValidator();
+
+        public FileListSettings()
+        {
+            BaseUrl = "https://filelist.io";
+            MinimumSeeders = IndexerDefaults.MINIMUM_SEEDERS;
+
+            Categories = new int[]
+            {
+                (int)FileListCategories.AUDIO,
```
Comment (Qstick) [inline src/NzbDrone.Core/Indexers/FileList/FileListSettings.cs]: Change the default to DOCS

File: src/NzbDrone.Core/HealthCheck/Checks/MonoNotNetCoreCheck.cs [hunk 27]
```diff
@@ -0,0 +1,57 @@
...
+            // Check for BSD
+            var output = _processProvider.StartAndCapture("uname");
+            if (output?.ExitCode == 0 && MonoUnames.Contains(output?.Lines.First().Content))
+            {
+                return new HealthCheck(GetType());
+            }
+
+            return new HealthCheck(GetType(),
+                                   HealthCheckResult.Warning,
+                                   "Please upgrade to the .NET Core version of Lidarr",
```
Comment (Qstick) [inline src/NzbDrone.Core/HealthCheck/Checks/MonoNotNetCoreCheck.cs]: Lidarr to Readarr

File: src/NzbDrone.Core/DecisionEngine/Specifications/RssSync/HistorySpecification.cs [hunk 28]
```diff
@@ -60,9 +60,9 @@ public virtual Decision IsSatisfiedBy(RemoteBook subject, SearchCriteriaBase sea
                         continue;
                     }
 
-                    // The author will be the same as the one in history since it's the same book.
-                    // Instead of fetching the author from the DB reuse the known author.
-                    var preferredWordScore = _preferredWordServiceCalculator.Calculate(subject.Author, mostRecent.SourceTitle);
+                    // The artist will be the same as the one in history since it's the same album.
+                    // Instead of fetching the artist from the DB reuse the known artist.
```
Comment (Qstick) [inline src/NzbDrone.Core/DecisionEngine/Specifications/RssSync/HistorySpecification.cs]: artist and album references, change these back

**Review by Qstick — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 5 above)
Comment (bakerboy448) [reply CONTRIBUTING.md]: drop this whole file; it's already updated

File: frontend/src/Components/Table/TableOptions/TableOptionsModal.js [hunk 29]
```diff
@@ -2,7 +2,7 @@ import _ from 'lodash';
 import PropTypes from 'prop-types';
 import React, { Component } from 'react';
 import { DndProvider } from 'react-dnd';
-import { HTML5Backend } from 'react-dnd-html5-backend';
+import HTML5Backend from 'react-dnd-html5-backend';
```
Comment (Qstick) [inline frontend/src/Components/Table/TableOptions/TableOptionsModal.js]: Revert this change, it needs the brackets

Comment (RetroDadson) [reply frontend/src/Components/Table/TableOptions/TableOptionsModal.js]: Incoming commit to fix this

File: src/NzbDrone.Update/UpdateEngine/InstallUpdateService.cs (hunk 19 above)
Comment (RetroDadson) [reply src/NzbDrone.Update/UpdateEngine/InstallUpdateService.cs]: Incoming commit

File: src/Lidarr.Api.V1/Blacklist/BlacklistBulkResource.cs (hunk 17 above)
Comment (RetroDadson) [reply src/Lidarr.Api.V1/Blacklist/BlacklistBulkResource.cs]: incoming commit

File: src/Lidarr.Api.V1/Blacklist/BlacklistBulkResource.cs (hunk 18 above)
Comment (RetroDadson) [reply src/Lidarr.Api.V1/Blacklist/BlacklistBulkResource.cs]: Incoming commit

File: src/NzbDrone.Core/HealthCheck/Checks/MonoNotNetCoreCheck.cs (hunk 27 above)
Comment (RetroDadson) [reply src/NzbDrone.Core/HealthCheck/Checks/MonoNotNetCoreCheck.cs]: Incoming commit

File: frontend/src/Settings/Profiles/Metadata/SecondaryTypeItems.js (hunk 12 above)
Comment (RetroDadson) [reply frontend/src/Settings/Profiles/Metadata/SecondaryTypeItems.js]: Incoming commit

File: frontend/src/Settings/Profiles/Metadata/SecondaryTypeItem.js (hunk 11 above)
Comment (RetroDadson) [reply frontend/src/Settings/Profiles/Metadata/SecondaryTypeItem.js]: Incoming commit

File: frontend/src/Settings/Profiles/Metadata/ReleaseStatusItems.js (hunk 10 above)
Comment (RetroDadson) [reply frontend/src/Settings/Profiles/Metadata/ReleaseStatusItems.js]: Incoming commit

File: frontend/src/Settings/Profiles/Metadata/ReleaseStatusItem.js (hunk 9 above)
Comment (RetroDadson) [reply frontend/src/Settings/Profiles/Metadata/ReleaseStatusItem.js]: Incoming commit

File: frontend/src/Settings/Profiles/Metadata/PrimaryTypeItems.js (hunk 8 above)
Comment (RetroDadson) [reply frontend/src/Settings/Profiles/Metadata/PrimaryTypeItems.js]: Incoming commit

File: frontend/src/Settings/Profiles/Metadata/PrimaryTypeItem.js (hunk 7 above)
Comment (RetroDadson) [reply frontend/src/Settings/Profiles/Metadata/PrimaryTypeItem.js]: Incoming commit

File: src/NzbDrone.Core/DecisionEngine/Specifications/RssSync/HistorySpecification.cs (hunk 28 above)
Comment (RetroDadson) [reply src/NzbDrone.Core/DecisionEngine/Specifications/RssSync/HistorySpecification.cs]: incoming commit

File: CONTRIBUTING.md (hunk 5 above)
Comment (RetroDadson) [reply CONTRIBUTING.md]: copied your latest file baker and inserted that

File: src/Readarr.Http/REST/RestModule.cs (hunk 4 above)
Comment (RetroDadson) [reply src/Readarr.Http/REST/RestModule.cs]: resolved

File: src/NzbDrone.Core/Datastore/Migration/041_add_indexer_and_enabled_release_profiles.cs (hunk 2 above)
Comment (RetroDadson) [reply src/NzbDrone.Core/Datastore/Migration/041_add_indexer_and_enabled_release_profiles.cs]: Incoming commit

File: src/NzbDrone.Core/Datastore/Migration/045_remove_chown_and_folderchmod_config.cs (hunk 3 above)
Comment (RetroDadson) [reply src/NzbDrone.Core/Datastore/Migration/045_remove_chown_and_folderchmod_config.cs]: Incoming commit

File: src/NzbDrone.Core/Notifications/SendGrid/SendGrid.cs (hunk 20 above)
Comment (RetroDadson) [reply src/NzbDrone.Core/Notifications/SendGrid/SendGrid.cs]: Commited

File: src/NzbDrone.Core/Notifications/Prowl/ProwlProxy.cs (hunk 21 above)
Comment (RetroDadson) [reply src/NzbDrone.Core/Notifications/Prowl/ProwlProxy.cs]: Commited

File: src/NzbDrone.Core/DecisionEngine/Specifications/RssSync/HistorySpecification.cs (hunk 28 above)
Comment (RetroDadson) [reply src/NzbDrone.Core/DecisionEngine/Specifications/RssSync/HistorySpecification.cs]: Commited

File: frontend/src/Styles/Variables/dimensions.js (hunk 16 above)
Comment (RetroDadson) [reply frontend/src/Styles/Variables/dimensions.js]: Commited

File: frontend/src/Store/Selectors/createHealthCheckSelector.js (hunk 15 above)
Comment (RetroDadson) [reply frontend/src/Store/Selectors/createHealthCheckSelector.js]: Commited

File: frontend/src/Store/Actions/bookFileActions.js (hunk 14 above)
Comment (RetroDadson) [reply frontend/src/Store/Actions/bookFileActions.js]: Commited

File: azure-pipelines.yml (hunk 6 above)
Comment (RetroDadson) [reply azure-pipelines.yml]: commit in; tests running so hopefully back in right

File: frontend/src/Store/Actions/authorEditorActions.js (hunk 13 above)
Comment (RetroDadson) [reply frontend/src/Store/Actions/authorEditorActions.js]: @Qstick this should become `bookFolder` right?

Comment (Qstick) [reply frontend/src/Store/Actions/authorEditorActions.js]: Maybe, I don't recall if we kept that option in Readarr. I'd have to look at the Author model

File: src/NzbDrone.Core/MediaFiles/BookImport/Specifications/SameFileSpecification.cs (hunk 22 above)
Comment (RetroDadson) [reply src/NzbDrone.Core/MediaFiles/BookImport/Specifications/SameFileSpecification.cs]: Commited

**Comment by RobinDadswell** [comment]
That’s alright I will have a look later on

File: frontend/src/Store/Actions/authorEditorActions.js (hunk 13 above)
Comment (RetroDadson) [reply frontend/src/Store/Actions/authorEditorActions.js]: That’s alright I will have a look later on

Comment (RetroDadson) [reply frontend/src/Store/Actions/authorEditorActions.js]: it wasn't in the model so I have removed it

File: src/NzbDrone.Core/Indexers/FileList/FileListSettings.cs (hunk 25 above)
Comment (RetroDadson) [reply src/NzbDrone.Core/Indexers/FileList/FileListSettings.cs]: Commited

File: src/NzbDrone.Core/Indexers/FileList/FileListSettings.cs (hunk 26 above)
Comment (RetroDadson) [reply src/NzbDrone.Core/Indexers/FileList/FileListSettings.cs]: Commited

File: src/NzbDrone.Core/Indexers/Newznab/NewznabCategoryFieldOptionsConverter.cs (hunk 24 above)
Comment (RetroDadson) [reply src/NzbDrone.Core/Indexers/Newznab/NewznabCategoryFieldOptionsConverter.cs]: commited

File: src/NzbDrone.Core/Indexers/Newznab/NewznabCategoryFieldOptionsConverter.cs (hunk 23 above)
Comment (RetroDadson) [reply src/NzbDrone.Core/Indexers/Newznab/NewznabCategoryFieldOptionsConverter.cs]: commited
